library/peerconnection.js

```diff
@@ -20,6 +20,11 @@ class CsioPeerConnection {
     modCommon.triggerEvent('newPeerConnection',
         {'userId': userId, 'pc': this.pc});
 
+    modCommon.triggerEvent('webrtcEvent',
+      {'pc': this.pc, 'eventLog': JSON.stringify(
+        {'userId': this.userId,'what': 'peerConnectionCreated'})
```

Comment:
would there be a better name than 'what'?

Reply:
Since, "peerConnectionCreated", "onDataChannelOpened" , "onDataChannelClosed", etc, currently are all events type, do you suggest "event" instead of "what" ? Or, we are adding application level log, so how about "api_event", or "app_event" ? Any suggestion ? Or I will go with "event".

Reply:
Please ignore my previous comment. I guess, if we implement what Karthik suggested, then there will be no key naming issue. For example for this event we can write -

``` "PeerConnection created for %{this.userId}" ```